fix(anomalies): anomaly revamp (#9465)
newrelic/docs-website: src/content/docs/alerts-applied-intelligence/applied-intelligence/anomaly-detection/anomaly-detection-applied-intelligence.mdx

  - /docs/alerts-applied-intelligence/applied-intelligence/proactive-detection/proactive-detection-new-relic-ai
  - /docs/alerts-applied-intelligence/applied-intelligence/proactive-detection/proactive-detection-applied-intelligence
---
import anomalydashboard from 'images/anomaly-dashboard.png'

import anomalydetails from 'images/anomaly-details.png'

import anomalyslacknotifications from 'images/anomaly-slack-notifications.png'

With applied intelligence's anomaly detection, New Relic alerts your team of any unusual behavior. New Relic observes your APM-monitored applications and understands baseline, or normal, performance. If behavior deviates from the baseline, we know right away and alert your team so you can address any errors promptly and efficiently. 

<img
  title="A screenshot of the anomaly dashboard in the New Relic UI"
  alt="A screenshot of the anomaly dashboard in the New Relic UI"
  src={anomalydashboard}
/>
<figcaption>The anomaly dashboard where your team can monitor any unusual behavior in your system.   
  </figcaption>

## How we use anomalies [#anomalies-in-use]

> At New Relic, we have our own developers who monitor the health of our own application. Our developers want to know if there are any outliers in our system's performance as soon as possible because we want to make sure that our customers are always able to access the data they need. New Relic's anomaly detection monitors three key golden signals: throughput, error rate and latency. Since we monitor our own application, New Relic developers always understand our baseline performance for these metrics. 

Reviewer: ```suggestion
At New Relic, our own developers monitor the health of our application. They want to know if there are any outliers in our system's performance as soon as possible because our customers should always have access to the data they need. New Relic's anomaly detection monitors three key golden signals: throughput, error rate and latency. Since we monitor our own application, New Relic developers always understand our baseline performance for these metrics. 
```

I shortened this excerpt to be shorter so it looks more scannable on the page. Didn't alter the actual writing, but just shifted it a bit. This is great framing btw. My only suggestion is to tighten the last sentence here, maybe? Something like "Since we monitor our own application, our developers use these metrics to XYZ" idk something like that to make it a tighter paragraph

> So, let's say that one afternoon there's a spike in response time and it's taking longer than usual for our customers to access the New Relic application. New Relic will flag this anomalous behavior because our latency metric data has deviated from its baseline. This doesn't necessarily mean there is a problem, it just indicates that New Relic has registered something out of the ordinary in our system and we should take a deeper look.

Reviewer: I did not understand this until I read your doc! (or what golden metrics were hah!)

import alertsAnomaliesSetUpperandLowerRanges from 'images/alerts_screenshot-full_anomalies-set-upper-and-lower-ranges.png'

import alertsFacetedAnomaliesTwo from 'images/alerts_screenshot-full_faceted-anomalies-two.png'

import alertsFacetedAnomalies from 'images/alerts_screenshot-full_faceted-anomalies.png'

import alertsConfigureYourAnomalyThresholds from 'images/alerts_screenshot-full_configure-your-anomaly-thresholds.png'

With applied intelligence's anomaly detection, New Relic alerts your team of any unusual behavior instantly. New Relic uses AI to constantly observe your applications. We use this information to determine your application's baseline, or expected, performance. Whenever behavior deviates from the baseline, we know right away and alert your team so you can address any errors promptly and efficiently. 

<img
  title="A screenshot of the anomaly dashboard in the New Relic UI"
  alt="A screenshot of the anomaly dashboard in the New Relic UI"
  src={anomalydashboard}
/>
<figcaption>The anomaly dashboard where your team can monitor any unusual behavior in your system.   
  </figcaption>

## How we use anomalies [#anomalies-in-use]

> At New Relic, our own developers know how important it is to monitor the health of our applications. We want our customers to access to the data they need whenever they need it so our team needs to be instantly alerted if there are any outliers in our system's performance. New Relic's anomaly detection uses applied intelligence to monitor three key golden signals: throughput, error rate and latency. With anomaly detection, our developers monitor the baseline performance for these metrics.

Reviewer 2: At New Relic, our own developers know how important it is to monitor the health of our applications. We want our customers to have access to the data they need and be instantly alerted if there are outliers in their system's performance. New Relic uses applied intelligence to automatically monitor changes in golden signal data for your APM entities.

  src={anomalyslacknotifications}
/>
<figcaption> How to set up slack or webhook notifications for anomalies in the New Relic UI.  
  </figcaption>

These events are also available for querying, creating custom dashboards, and alerting. After we set up an anomaly detection configuration (a group of apps we're interested in), we can add this configuration as a source. Then the anomalies will be automatically correlated with other data sources via incident intelligence.

## There are two types of anomaly detection: automatic and custom

[Automatic anomalies](/docs/alerts-applied-intelligence/applied-intelligence/anomaly-detection/anomaly-detection-applied-intelligence#automatic-anomalies) are the most efficient way for your team to learn about unusual behavior in your APM-monitored applications. Automatic anomaly detection is a hands-off tool your team can implement to ensure that you're notified the moment behavior in your application deviates from baseline. You can use automatic anomalies to identify the source of the problem and take the appropriate steps to get your system running smoothly again.

> [Custom anomalies](/docs/alerts-applied-intelligence/applied-intelligence/anomaly-detection/anomaly-detection-applied-intelligence#custom-anomalies) allow increased configurability for your team. While custom anomalies require more initial setup than our automatic anomaly option, custom anomalies offer more settings so your team can better control of what signals your team is detecting. Custom anomalies also allow your team the capability to alert on any NQRL condition.

Reviewer 2: [Custom anomalies](/docs/alerts-applied-intelligence/applied-intelligence/anomaly-detection/anomaly-detection-applied-intelligence#custom-anomalies) allow increased configurability for your team.  Custom anomalies provide your team with the capability to alert on any NQRL condition and to adjust and optimize your thresholds. Custom anomalies also have all the same advanced tuning settings as static alerting to ensure you are seeing just those anomaly incidents important to you.

Author: done!

Applications will not always generate anomalies, so it can be normal to not receive any detections.

### Requirements [#requirements]

To use New Relic's anomaly detection, ensure you have:

* An APM agent installed for at least one application.
* To receive notifications in Slack, you'll need to ask your IT administrator to install the New Relic application in [your Slack workspace](https://slack.com/help/articles/202035138-Add-an-app-to-your-workspace).

For more on data limits, see [Data limits](#data-limits).

### Set up notifications for anomaly detection [#set-up]

> Proactive detection is enabled automatically, at no additional cost. To receive notifications or to have a configuration (group of apps) that you can add as a source for incident intelligence, you will need to create a anomaly detection configuration. You can create a configuration in the anomaly detection UI:

Reviewer 2: replaces proactive detection with "automatic anomaly detection".  May want to do a quick search for proactive detection across all our docs and replace with "automatic anomaly detection".

Author: done!

  </tbody>
</table>

### Add anomalies as source in incident intelligence [#add-source]

By integrating incident intelligence with your anomaly detection, you can get context and correlations. To learn about doing this in incident intelligence, see [Configure sources](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/get-started-incident-intelligence/#1-configure-sources).

You can also select **Connect to incident intelligence** from inside of a configuration.

### Webhook payload and examples [#webhook]

> Proactive detection sends the event body in JSON format via HTTPS POST. The system expects the endpoint to return a successful HTTP code (2xx). If you use webhooks to configure Proactive detection, use these examples of the webhook body format and JSON schema.

Reviewer 2: Replace "proactive detection" with automatic anomalies

Author: done!
